src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -352,6 +366,19 @@ public boolean isAllowedTargetBranch() {
         return true;
     }
 
+    private GitUser getPRCommitAuthor (){
+        try {
```

Comment:
This looks like it'll return the author of the latest commit, no? That's ok for a first version of this feature, I think. But could you add a comment in the code...something like `\\Returns author of latest commit` ?